### ISSUE TITLE
[Bug]: Cannot Load Providers

### ISSUE THREAD

**Issue by 117ksk** [bushvin/actor-export#10]
### Foundry VTT version

11.315

### Game system

pf2e

### Game system version

5.15.3

### actor-export module version

2024.5.5

### What browsers are you seeing the problem on?

Firefox, Chrome

### What Provider is affected?

Bushvin's modified PF2e Remaster Character sheet, Bushvin's PF2e Remaster Character sheet (deprecated), PF2e Remaster Character sheet (original), PF2e Scribe Generator

### What happened?

When any provider is selected to be used for the download, an error pops up and nothing executes or downloads.

### Relevant log output

```Shell
An error ocurred executing the 'pf2e-scribe' provider: error loading dynamically imported module
Uncaught (in promise) Error: TypeError: error loading dynamically imported module
```

### Sample Actor

```JSON
N/A
```

### Attachments

N/A

**Comment by 117ksk** [comment]
Seems to be issue with how my instance is hosted and not related to the module itself. Will close.

**Comment by bushvin** [comment]
Would you mind sharing why you came to this conclusion? Maybe it's worth documenting in my wiki

**Comment by 117ksk** [comment]
To be honest, I came to this conclusion as a guess. I have two foundry instances running on a windows server as node services and this module failed with the above errors. When I tried it on a local windows foundry application instance, it worked fine. I use the same modules on the server instances that I do on my local install. No idea what the difference would be other than how the instances are being ran.

I also ended up exporting the exact actor that I failed to use this module on from the node server instance and importing into my local install. So same exact actor data, same modules, same version etc, only thing was native windows vs node js.
